app/src/main/java/com/google/jetpackcamera/MainActivity.kt

```diff
@@ -81,6 +84,11 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
         var uiState: MainActivityUiState by mutableStateOf(Loading)
 
+        // start trace between app starting and the earliest possible completed capture
+        // todo: only run traces on a specific build version or flavor(?)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
There shouldn't be a need to limit it. Traces are essentially no-ops when atrace isn't turned on. They're also very lightweight otherwise.

If this is due to API level restrictions of `android.os.Trace`, we should use `androidx.tracing` instead.